internal/repos/subscription/db_repo_test.go

```diff
@@ -31,7 +31,7 @@ func TestCreateSubscription_Success(t *testing.T) {
 	assert.NoError(t, err)
 	defer closeDB(mock, db, t)
 
-	repo := repos.NewSubscriptionDBRepo(db)
+	repo := subr.NewSubscriptionDBRepo(db)
```

Comment:
```subr.NewSubscriptionDBRepo``` is tautology, better name is 

```suggestion
	repo := subr.NewDbRepo(db)
```